src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
+import { Row, Col } from 'antd'
 
 function App() {
+  const [allFoods, setAllFoods] = useState(foods)
+  
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...allFoods]
+    setAllFoods(updatedFood)
+  }
+
+  const searchFood = (filterfoods) => {
+    const searchedFood = allFoods.filter(food => 
+    food.name.toLowerCase().includes(filterfoods.toLowerCase()))
+    setAllFoods(searchedFood)
```

Comment:
The `allFoods` will be the original array that will always contain every food. So here you want to update the `setShowFoods` instead of the `setAllFoods` like this:
`setShowFoods(searchedFood)`